lib/stove/community.rb

```diff
@@ -86,9 +86,10 @@ def yank(name)
     #
     def connection
       @connection ||= ChefAPI::Connection.new do |conn|
-        conn.endpoint = ENV['STOVE_ENDPOINT'] || Config.endpoint || DEFAULT_ENDPOINT
-        conn.client   = ENV['STOVE_USERNAME'] || Config.username
-        conn.key      = ENV['STOVE_KEY']      || Config.key
+        conn.endpoint   = ENV['STOVE_ENDPOINT'] || Config.endpoint || DEFAULT_ENDPOINT
+        conn.client     = ENV['STOVE_USERNAME'] || Config.username
+        conn.key        = ENV['STOVE_KEY']      || Config.key
+        conn.ssl_verify = ENV['STOVE_SSL_VERIFY'].nil? ? Config.ssl_verify : ENV['STOVE_SSL_VERIFY'].downcase == 'true'
```

Comment:
This still feels weird to me. The default is "true", so the env option should be to disable it:

```
STOVE_NO_SSL_VERIFY=1
```

If it's set, it's set. We shouldn't worry about true/false/yes/no/1/0 :smile: